autoload/eft.vim

```diff
@@ -65,7 +65,7 @@ function! s:goto(repeat, dir, till) abort
   let l:curpos = getcurpos()
   let l:line = getline(l:curpos[1])
   let l:col = l:curpos[2]
-  let l:indices = a:dir ==# s:Dir.Next ? range(l:col, strlen(l:line)) : range(l:col - 2, 0, -1)
+  let l:indices = a:dir ==# s:Dir.Next ? range(l:col + (a:till ? 1 : 0), strlen(l:line)) : range(l:col - (a:till ? 3 : 2), 0, -1)
```

Comment:
Thank you for your contribution.
I think the `a:til` should be `a:till`.

And I think we should add the `let l:tilloff = a:till ? 1 : 0` first.